swagger.yaml

```diff
@@ -149,6 +165,24 @@ definitions:
               lastName:
                 type: string
                 description: Preferred last name
+              previousRecords:
+                type: array
+                description: Previous records associated with the person
+                items:
+                  properties:
+                    osuID:
+                      type: string
+                      description: 9 digit OSU ID
+                      example: "931234567"
+                    firstName:
+                      type: string
+                      description: Previous first name
+                    middleName:
+                      type: string
+                      description: Previous middle name
+                    lastName:
+                      type: string
+                      description: Previous last name
```

Comment:
I forgot did we discuss this in person or not. Could we also add the modification date for each record?